core/src/test/java/hudson/FilePathTest.java

```diff
@@ -641,7 +644,7 @@ public void testValidateCaseSensitivity() throws Exception {
             when(con.getInputStream())
               .thenReturn(someZippedContent());
 
-            assertTrue(d.installIfNecessaryFrom(url, null, null));
+            assertTrue(d.installIfNecessaryFrom(url, null, ""));
```

Comment:
Does not seem to be related

Reply:
The annotation has been introduced by @jglick in https://github.com/jenkinsci/jenkins/blame/09bcc5d6538b3cfffbf71228ebd1679e3e20d8b2/core/src/main/java/hudson/FilePath.java#L766 3 years ago, Does not matter now IMHO, I am fine with fixing it in this PR

Reply:
True, it is not directly related - however the test case fails without the change.

Reply:
@daniel-beck @jglick @dwnusbaum do we care about that? Maybe we need to do some extra NPE checks